spec/models/like_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before :each do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+
+    @post = Post.create(author: @post, title: 'Hello', text: 'This is my first post')
+
+    @comment = Comment.create(post: @comment, author: @user, text: 'Hi Tom!')
+
+    @like = Like.create(author: @author, post: @post)
+  end
+
+  it 'likesCounter wi' do
```

Comment:
[OPTIONAL] It's great you added a test for likesCounter. However, I couldn't catch the name of the test..., I would suggest adding a clear description to your test :).